app/views/idv/verify_errors/show.html.erb

```diff
@@ -0,0 +1,41 @@
+<%= render(
+      'idv/shared/error',
+      title: t('titles.failure.information_not_verified'),
+      heading: t('idv.failure.verify.heading'),
+    ) do %>
+    <p>
+      <% if decorated_session.sp_name.present? %>
+        <%= link_to(
+              t('idv.failure.verify.fail_link_html', sp_name: decorated_session.sp_name),
```

Comment:
The text says this is a help link, but it returns to the same failure to proof path as the Exit Login.gov button. I see `idv.troubleshooting.options.get_help_at_sp` in other places, maybe that's the right link?